app/pb_kits/playbook/pb_popover/_popover.js

```diff
@@ -0,0 +1,54 @@
+import { createPopper } from '@popperjs/core'
+
+class Popover {
+  constructor(
+    button = '#button',
+    tooltip = '#tooltip',
+    placement = 'left',
+    offset = ''
+  ) {
+    this.button = button
+    this.tooltip = tooltip
+    this.placement = placement
+    this.offset = offset
+    this.setupPopper()
+  }
+
+  //getters
+  get popoverButton() {
```

Comment:
What if this is used for tool tips or other pop-able elements. Thinking maybe this wouldn’t be a button. Possibly rename?

Reply:
Any suggestions? I was thinking about 'Target' but that carries some possible meaning already in javascript... not sure.

Reply:
getpopoverTriggerElement() might be a little more generic for us here. 
It scales better-- no matter what the element is. Thoughts?